.github/workflows/mkdocs.yml

```diff
@@ -30,4 +31,6 @@ jobs:
       - name: Install dependencies
         run: poetry install --no-interaction --no-ansi
       - name: Update MkDocs
-        run: poetry run task mkdocs
\ No newline at end of file
+        run: |
+          git pull
```

Comment:
Use `fetch-depth: 0` in the options for the `checkout` action to avoid needing to run `git pull`; see the [documentation](https://github.com/actions/checkout#usage)

Reply:
Didn't know this existed. Added. Thanks!